jib-core/build.gradle

```diff
@@ -1,69 +1,20 @@
 plugins {
-  id 'java-library'
-  id 'checkstyle'
-  id 'com.github.sherter.google-java-format' version '0.8'
-  id 'net.ltgt.apt' version '0.19'
-  id 'net.ltgt.errorprone' version '0.6'
-  id 'biz.aQute.bnd.builder' version '4.0.0'
-
-  // For release.
-  id 'maven'
-  id 'net.researchgate.release' version '2.7.0'
-}
-
-group 'com.google.cloud.tools'
-
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
-compileJava.options.encoding = 'UTF-8'
-compileJava.options.compilerArgs += [ '-Xlint:deprecation' ]
-compileTestJava.options.compilerArgs += [ '-Xlint:deprecation' ]
-
-repositories {
-  mavenCentral()
-}
-
-sourceSets {
-  integrationTest {
-    java.srcDir file('src/integration-test/java')
-    resources.srcDir file('src/integration-test/resources')
-  }
-}
-
-configurations {
-  integrationTestImplementation.extendsFrom testImplementation
-  integrationTestRuntime.extendsFrom testRuntime
+  id 'net.researchgate.release'
+  id 'maven-publish'
 }
 
 dependencies {
-  // Make sure these are consistent with jib-maven-plugin.
-  implementation 'com.google.http-client:google-http-client:1.31.0'
-  implementation 'com.google.http-client:google-http-client-apache-v2:1.31.0'
-  implementation 'org.apache.commons:commons-compress:1.18'
-  implementation 'com.google.guava:guava:27.0.1-jre'
-  implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.9.2'
-  implementation 'org.ow2.asm:asm:7.0'
-
-  testImplementation 'junit:junit:4.12'
-  testImplementation 'org.mockito:mockito-core:2.23.4'
-  testImplementation 'org.slf4j:slf4j-api:1.7.25'
-  testImplementation 'com.github.stefanbirkner:system-rules:1.19.0'
+  implementation "com.google.http-client:google-http-client:${dependencyVersions.GOOGLE_HTTP_CLIENT}"
+  implementation "com.google.http-client:google-http-client-apache-v2:${dependencyVersions.GOOGLE_HTTP_CLIENT_APACHE_V2}"
+  implementation "org.apache.commons:commons-compress:${dependencyVersions.COMMONS_COMPRESS}"
```

Comment:
[#1902](https://github.com/GoogleContainerTools/jib/pull/1902/files#diff-2c8a5b94f9299ef70d5acb27fae2bfd8R46-R47) adds `google-auth-library-oauth2-http` and upgrades Guava version. Can you update them here?

Reply:
Ah, wait. I think it isn't required in `jib-core` but in `jib-plugins-common`. Let me check this again.

Reply:
will update when it merges

Reply:
Wrong. It is required in `jib-core`. Can you add it and upgrade Guava?